detect_variants/detect_variants.cwl

```diff
@@ -27,6 +27,9 @@ inputs:
         secondaryFiles: [.tbi]
     strelka_exome_mode:
         type: boolean
+    strelka_cpu_num:
```

Comment:
should this be something like `strelka_cpu_reserved` or `min_strelka_cpus`? Do you want to go ahead and parameterize the number of threads/CPU strelka *thinks* it's using in the command line?